haskell/toolchain.bzl

```diff
@@ -0,0 +1,192 @@
+HaskellPackageInfo = provider()
```

Comment:
Is this just a stub? I'm not sure what it does if this provider has no arguments.